Update gameobject_questitem.md (#110)
azerothcore/wiki: docs/gameobject_questitem.md

Column | Type | Description
--- | --- | ---

> GameObjectEntry | int(10) unsigned | Entry ID of a gameobject that contains quest items only while the related quest is active.

Reviewer: ```suggestion
GameObjectEntry | int(10) unsigned | Entry ID of a gameobject that contains quest items when the related quest is active.
```

Author: better, although the rest of them say "only while"

Reviewer: well you can edit those too :D but thank you for PRs!!

to edit directly in github you can go here https://github.com/azerothcore/wiki/pull/110/commits then click the "..." and "edit file" and you " Commit directly to the patch-3 branch. "

Author: nice, thanks for the tip
